### ISSUE TITLE
Error while running Truvari

### ISSUE THREAD

**Issue by prasundutta87** [spiralgenetics/truvari#84]
**Describe the bug**
Hi, 

I am using Truvari v3.0-dev and while running, I am getting an error in the form of KeyError: 'pct'. Although, I am getting the metrics I want, I am not getting anything under # State by SizeCat and SVTYPE in giab_report.txt.

**To Reproduce**
1.  Using 'Truvari v3.0-dev'
2. Run 'truvari bench -b ../../Reference_SV_sets/HG002_SVs_Tier1_v0.6.vcf.gz -c GM24385_minimap2_cuteSV_DUP_to_INS.vcf.gz --passonly --refdist 1000 --giabreport -p 0.00 --includebed HG002_SVs_Tier1_v0.6.bed -f hs37d5.fa -o truvari_minimap2_cuteSV_DUP_INS'
3. See '2021-10-01 11:44:12,393 [INFO] Optimizing memory
2021-10-01 11:44:12,652 [INFO] Optimized 14.96MB to 11.36MB
2021-10-01 11:44:12,903 [INFO] Finished vcf2df
Traceback (most recent call last):
  File "/home/u027/project/software/conda/envs/SGP2/lib/python3.8/site-packages/pandas/core/indexes/base.py", line 3080, in get_loc
    return self._engine.get_loc(casted_key)
  File "pandas/_libs/index.pyx", line 70, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/index.pyx", line 101, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/hashtable_class_helper.pxi", line 4554, in pandas._libs.hashtable.PyObjectHashTable.get_item
  File "pandas/_libs/hashtable_class_helper.pxi", line 4562, in pandas._libs.hashtable.PyObjectHashTable.get_item
KeyError: 'pct'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/u027/project/software/conda/envs/SGP2/lib/python3.8/site-packages/pandas/core/generic.py", line 3826, in _set_item
    loc = self._info_axis.get_loc(key)
  File "/home/u027/project/software/conda/envs/SGP2/lib/python3.8/site-packages/pandas/core/indexes/base.py", line 3078, in get_loc
    casted_key = self._maybe_cast_indexer(key)
  File "/home/u027/project/software/conda/envs/SGP2/lib/python3.8/site-packages/pandas/core/indexes/category.py", line 496, in _maybe_cast_indexer
    return self._data._unbox_scalar(key)
  File "/home/u027/project/software/conda/envs/SGP2/lib/python3.8/site-packages/pandas/core/arrays/categorical.py", line 1721, in _unbox_scalar
    code = self.categories.get_loc(key)
  File "/home/u027/project/software/conda/envs/SGP2/lib/python3.8/site-packages/pandas/core/indexes/base.py", line 3082, in get_loc
    raise KeyError(key) from err
KeyError: 'pct'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/u027/project/software/conda/envs/SGP2/bin/truvari", line 8, in <module>
    sys.exit(main())
  File "/home/u027/project/software/conda/envs/SGP2/lib/python3.8/site-packages/truvari/__main__.py", line 64, in main
    TOOLS[args.cmd](args.options)
  File "/home/u027/project/software/conda/envs/SGP2/lib/python3.8/site-packages/truvari/bench.py", line 539, in bench_main
    truvari.make_giabreport(args, outputs["stats_box"])
  File "/home/u027/project/software/conda/envs/SGP2/lib/python3.8/site-packages/truvari/giab_report.py", line 50, in make_giabreport
    view["pct"] = view["tpbase"] / view.sum(axis=1)
  File "/home/u027/project/software/conda/envs/SGP2/lib/python3.8/site-packages/pandas/core/frame.py", line 3163, in __setitem__
    self._set_item(key, value)
  File "/home/u027/project/software/conda/envs/SGP2/lib/python3.8/site-packages/pandas/core/frame.py", line 3243, in _set_item
    NDFrame._set_item(self, key, value)
  File "/home/u027/project/software/conda/envs/SGP2/lib/python3.8/site-packages/pandas/core/generic.py", line 3829, in _set_item
    self._mgr.insert(len(self._info_axis), key, value)
  File "/home/u027/project/software/conda/envs/SGP2/lib/python3.8/site-packages/pandas/core/internals/managers.py", line 1197, in insert
    new_axis = self.items.insert(loc, item)
  File "/home/u027/project/software/conda/envs/SGP2/lib/python3.8/site-packages/pandas/core/indexes/extension.py", line 379, in insert
    code = arr._validate_scalar(item)
  File "/home/u027/project/software/conda/envs/SGP2/lib/python3.8/site-packages/pandas/core/arrays/categorical.py", line 1251, in _validate_fill_value
    raise TypeError(
TypeError: 'fill_value=pct' is not present in this Categorical's categories
'

**Expected behavior**
I am not sure why the error is coming. Seems like a python specific error, but I am not exactly sure.

**Example Data**
If applicable/possible, add example data to help recreate your problem.

**Additional context**

**Comment by ACEnglish** [comment]
I think the first thing to try is for you to move to 3.1.0. In the [versioning docs](https://github.com/spiralgenetics/truvari/wiki/Development#versioning) we mention how `*-dev` are development branches. Work in development is more likely to have bugs like this.
If that doesn't work, let me know and we can dig in further.

**Comment by prasundutta87** [comment]
Thanks a lot for the quick reply. I can see that GitHub still has the version v3.0.0 as the latest version released 6 days ago. Where can I access v3.1.0?

**Comment by ACEnglish** [comment]
My mistake - I meant to say use version 3.0.0.  The easiest way to get it is to take advantage of our pypi distribution
```
python3 -m pip install truvari==3.0.0
```

Alternatively, if you'd like to build from the repository, after pulling the code
```
git checkout tags/v3.0.0
python3 setup.py install
```

Note if you have already pulled the repository, you'll need to update your local copy via:
```
git pull
git fetch --all --tags
```
And then checkout the v3.0.0 tag.

If you'd like to test 3.1.0-dev (which is the current development branch), see [Development](https://github.com/spiralgenetics/truvari/wiki/Development) for more details.

**Comment by prasundutta87** [comment]
Thanks a lot for this, Adam. I installed it in my conda environment, if I remember correctly, using pip itself as bioconda still had a very old version. Let me check how do I remove or update it.

**Comment by prasundutta87** [comment]
Hi Again!

Just tried v3.0.0. The error still exists and "State by SizeCat and SVTYPE" is still empty.

**Comment by ACEnglish** [comment]
Okay, looking more closely at your error, I think one possible problem may be with the pandas versioning.

Could you let me know what version of pandas is installed with truvari? `pip freeze | grep -i pandas`?

**Comment by prasundutta87** [comment]
It is v1.2.4.

**Comment by ACEnglish** [comment]
Yep, that's the problem. In v3.0.0 the requirements incorrectly request pandas>=1.0.5. Below is how I was about to recreate your error and fix it. 

tl;dr; Update pandas with `python3 -m pip install pandas>=1.3.3` . I will work on cutting a hot-fix v3.0.0 named v3.0.1

I can recreate the error by running the following in a truvari v3.1.0-dev docker container
```
python3 -m pip install pandas==1.2.4 --force
bash repo_utils/truvari_ssshtests.sh
```
This makes 7 errors all releated to pandas for `test_bench_giab`, `test_bench_giab_report`, `test_vcf2df_result`, `test_vcf2df_dir_result`, `test_vcf2df_single`, `test_vcf2df_subset`, `test_vcf2df_multi`. This includes the `TypeError: 'fill_value=pct' is not present in this Categorical's categories`

This can be resolved by resetting the docker container and using the newer pandas version
```bash
docker run -v `pwd`:/data --entrypoint /bin/bash -it truvari
python3 -m pip freeze | grep -i pandas
# pandas==1.3.3
bash repo_utils/truvari_ssshtests.sh
# ...
# sshtest v0.1.5
# 
# 74        Tests
# 0         Failures
# 74        Successes
```
Therefore, pandas needs to be updated to `>=1.3.3`

**Comment by prasundutta87** [comment]
Thanks a lot for this, Adam! :-)
